### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by origamiserviceuser**
:robot: I have created a release *beep* *boop*
---

<details><summary>ft-concept-button: 1.2.2</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>n-notification: 8.2.5</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-autocomplete: 1.8.1</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-banner: 4.5.1</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-big-number: 3.2.2</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-buttons: 7.8.3</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-comments: 10.3.2</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-cookie-message: 6.6.1</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-editorial-layout: 2.4.2</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-editorial-typography: 2.3.5</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-footer: 9.2.8</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-footer-services: 4.2.7</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-forms: 9.11.3</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-header: 11.0.7</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-header-services: 5.5.1</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-icons: 7.6.1</summary>

### ⚠ BREAKING CHANGES

* create o-multi-select component

### Features

* create o-multi-select component ([5730a3a](https://github.com/Financial-Times/origami/commit/5730a3a13399b2caec6bf1dffdb9f8d7c2b35ada))

### Miscellaneous Chores

* o-icons, release 7.6.0 not 8.0.0 ([#1240](https://github.com/Financial-Times/origami/issues/1240)) ([cb03604](https://github.com/Financial-Times/origami/commit/cb03604495d6bc34de2d49ceafc5bd46b76d9eae))
* o-icons, update support matrix ([#1243](https://github.com/Financial-Times/origami/issues/1243)) ([27e32c3](https://github.com/Financial-Times/origami/commit/27e32c393eed4070d21760cbfe9563cae4c94566))
</details>

<details><summary>o-labels: 6.5.7</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-layout: 5.3.3</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-message: 5.4.3</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-meter: 3.2.3</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-multi-select: 2.2.1</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-overlay: 4.2.9</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-quote: 5.3.3</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-share: 10.0.0</summary>

### ⚠ BREAKING CHANGES

* Replace Twitter with X [OR-282] ([#1239](https://github.com/Financial-Times/origami/issues/1239))

### Features

* Replace Twitter with X [OR-282] ([#1239](https://github.com/Financial-Times/origami/issues/1239)) ([4561d0d](https://github.com/Financial-Times/origami/commit/4561d0d40dd2ae513f2a1de4eee7456e14d63b40))

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-social-follow: 1.0.8</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-spacing: 3.2.4</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-stepped-progress: 4.0.8</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-subs-card: 6.2.5</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-table: 9.3.2</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-teaser: 6.2.8</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-teaser-collection: 4.2.4</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-tooltip: 5.3.1</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-topper: 6.0.7</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

<details><summary>o-video: 7.2.10</summary>

### Bug Fixes

* update o-typography dependency  ([fb45b47](https://github.com/Financial-Times/origami/commit/fb45b47274241ea828f7dd50233441a76a215a51))
</details>

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

[OR-282]: https://financialtimes.atlassian.net/browse/OR-282?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: components/o-icons/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+### Features
+
+* create o-multi-select component ([5730a3a](https://github.com/Financial-Times/origami/commit/5730a3a13399b2caec6bf1dffdb9f8d7c2b35ada))
+
+
+### Miscellaneous Chores
+
+* o-icons, release 7.6.0 not 8.0.0 ([#1240](https://github.com/Financial-Times/origami/issues/1240)) ([cb03604](https://github.com/Financial-Times/origami/commit/cb03604495d6bc34de2d49ceafc5bd46b76d9eae))
+* o-icons, update support matrix ([#1243](https://github.com/Financial-Times/origami/issues/1243)) ([27e32c3](https://github.com/Financial-Times/origami/commit/27e32c393eed4070d21760cbfe9563cae4c94566))
+
```
Comment (notlee) [inline components/o-icons/CHANGELOG.md:19]:
```suggestion
### Miscellaneous Chores

* o-icons, update support matrix ([#1243](https://github.com/Financial-Times/origami/issues/1243)) ([27e32c3](https://github.com/Financial-Times/origami/commit/27e32c393eed4070d21760cbfe9563cae4c94566))

```

**Review by notlee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/ft-concept-button-v1.2.2 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/n-notification-v8.2.5 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-autocomplete-v1.8.1 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-banner-v4.5.1 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-big-number-v3.2.2 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-buttons-v7.8.3 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-comments-v10.3.2 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-cookie-message-v6.6.1 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-editorial-layout-v2.4.2 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-editorial-typography-v2.3.5 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-footer-v9.2.8 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-footer-services-v4.2.7 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-forms-v9.11.3 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-header-v11.0.7 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-header-services-v5.5.1 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-icons-v7.6.1 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-labels-v6.5.7 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-layout-v5.3.3 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-message-v5.4.3 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-meter-v3.2.3 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-multi-select-v2.2.1 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-overlay-v4.2.9 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-quote-v5.3.3 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-share-v10.0.0 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-social-follow-v1.0.8 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-spacing-v3.2.4 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-stepped-progress-v4.0.8 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-subs-card-v6.2.5 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-table-v9.3.2 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-teaser-v6.2.8 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-teaser-collection-v4.2.4 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-tooltip-v5.3.1 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-topper-v6.0.7 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release is at https://github.com/Financial-Times/origami/releases/tag/o-video-v7.2.10 :sunflower:

**Pull request merged**